boards/v2/qemu/nios2/Kconfig.qemu_nios2

```diff
@@ -1,6 +1,6 @@
 # SPDX-License-Identifier: Apache-2.0
 
 config BOARD_QEMU_NIOS2
-	bool "QEMU NIOS II target"
-	depends on SOC_NIOS2_QEMU
+	bool
+	select SOC_NIOS2_QEMU
 	select QEMU_TARGET
```

Comment:
`select QEMU_TARGET` should go to the `Kconfig` file, not `Kconfig.<board>`.

Reply:
Fixed